src/guides/v2.4/extension-dev-guide/indexing.md

```diff
@@ -107,11 +107,14 @@ return [
 ];
 ```
 
-In case of a failure, this mode will return the correct status of the indexer.
+In case of a failure during the reindexing of a certain indexer, this mode will return a more accurate status of the indexer.
+The status can be obtained from the indexer grid in the Magento Admin or through the `bin/magento indexer:status` command in the CLI.
 
-The current status can be obtained from the indexer grid in Magento Admin or through the index status in the CLI.
+{:.bs-callout-warning}
+The status values in the `indexer_state` or `mview_state` database tables may not be the same as what you are seeing because they sometimes do not get updated when an indexer fails.
```

Comment:
```suggestion
The status values in the `indexer_state` or `mview_state` database tables may not be the same as what is observed, because they sometimes do not get updated when an indexer fails.
```